requirements.txt

```diff
@@ -1,40 +1,20 @@
-happybase==0.7      --hash=sha256:95865c4aebad370c538907278520fee13e760be67c71eb989a7a9fb0d266fe90
 isodate==0.5.4 --hash=sha256:42105c41d037246dc1987e36d96f3752ffd5c0c24834dd12e4fdbe1e79544e31
-pika==0.10.0      --hash=sha256:63131aaeec48a6c8f1db1fe657e1e74cf384c3927eb7d1725e31edae4220dea4      --hash=sha256:7277b4d12a99efa4058782614d84138983f9f89d690bdfcea66290d810806459
 psycopg2==2.6.1      --hash=sha256:6acf9abbbe757ef75dc2ecd9d91ba749547941abaffbe69ff2086a9e37d4904c
-pyhs2==0.6.0      --hash=sha256:16aef41cc4e3d9a4a3e3a597481451aa8b26ba3e68bda2c820b1fef708206e21
-requests==2.9.1 --hash=sha256:113fbba5531a9e34945b7d36b33a084e8ba5d0664b703c81a7c572d91919a5b8 --hash=sha256:c577815dd00f1394203fc44eb979724b098f88264a9ef898ee45b8e5e9cf587f
 SQLAlchemy==0.9.8      --hash=sha256:de3ac04ecab0a7e0af64eee30a71e723ba5886b958205f7354dd62439da60389
 statsd==2.1.2      --hash=sha256:45db168312a6820fd78697c7b6fd0196f3af922c8ebc88540edd0d6fafed554d
-thrift==0.8.0      --hash=sha256:80b3c0c8649ce590bd25871550f6670eabe2e98a5d1ef5e40f92b88597d8081d
 ujson==1.35      --hash=sha256:f66073e5506e91d204ab0c614a148d5aa938bdbf104751be66f8ad7a222f5f86
 
 
 # deps of deps, pinned here for --require-hashes mode
 configman==1.2.11 --hash=sha256:46c8eb10e37b3df041789570699e9b460ca22e6dfd256ca20e4204db0cfb2bc5 --hash=sha256:16798e8a67467f50f8f9b080c8e70a41f0ff7d2dd061e74423e53bc7ed92fb8d
 configobj==4.7.2 --hash=sha256:515ff923462592e8321df8b48c47e3428f8d406ee22b8de77bef969d1af11171
 contextlib2==0.4.0 --hash=sha256:55a5dc78f7a742a0e756645134ffb39bbe11da0fea2bc0f7070d40dac208b732
-decorator==4.0.6 --hash=sha256:cc3dcda7835a7eafa887fad9d6a69b2d5fd77c0b1c2a1608f52b48e345e37206 --hash=sha256:1c6254597777fd003da2e8fb503c3dbf3d9e8f8d55f054709c0e65be3467209c
-Mako==0.7.3     --hash=sha256:49842441d24d88d0ecfc38686d021cb3b1bee7abfc541e9fc7345e4620deeab6
-MarkupSafe==0.15 --hash=sha256:339ec04d20ad9cdccbfe9f38dce6182cc504ce789e5d0f9647eaa752f0f95300
 mccabe==0.3.1 --hash=sha256:bd6c080fb372aebcb0ce19e35ddac744f2abf5a7befa207db2d1097d48efe63a --hash=sha256:5f7ea6fb3aa9afe146d07fd6d5cedf788747d8b0c29e44732453c2b2db1e3d16
-python-editor==0.5 --hash=sha256:f65c033ede0758663b9ff6a29d702f0b09198ad7c4ef96c9d37ccdfbf7bbf6fa --hash=sha256:1184f947a0f5cfa0f9528766280ac90e8a07cbf43bedda17a4672e22cf91794e
-raven==5.9.0 --hash=sha256:362cca6f296e114960463ba2860e1e5d70c41a6737389eb313bde614171371aa --hash=sha256:c545c8bb603bd0a7373e9760c6fb16bb0657d6f356ed1210cbb94ce418f373be
-sasl==0.1.3 --hash=sha256:4f324226e1ee85ac41520579ba9de703294012602e536217fd0e7456e147d886
-simplejson==3.6.5 --hash=sha256:2a3189f79d1c7b8a2149a0e783c0b4217fad9b30a6e7d60450f2553dc2c0e57e
-urllib3==1.9.1 --hash=sha256:410bea96df2479fa33986c40c3eefa9c1e7ba95a7f70711c315860bbe2d66c7f --hash=sha256:d858379ef5988d4534bb8909432d697422100aaff272299d661339836b6dae9b
 
 
 # test deps
```

Comment:
If we were good people, we'd split this out into a `requirements-dev.txt` so that these don't end up being required (and the very specific versions, no less) by the final socorrolib package.